__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data")
+
+beforeEach(() => { return seed (data) });
+afterAll(() => { return db.end() });
+
+describe("/api/topics", () => {
+    it("GET - 200, returns an array of all topics", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
```

Comment:
You are missing an assertion here to check the length of the body. If the body is an empty array, the assertions on line 17/18 won't run and the test will pass when it shouldn't! 🫨